skipper.go

```diff
@@ -503,11 +503,10 @@ type Options struct {
 	OIDCSecretsFile string
 
 	// API Monitoring feature is active (feature toggle)
-	ApiUsageMonitoringEnable                       bool
-	ApiUsageMonitoringRealmKeys                    string
-	ApiUsageMonitoringClientKeys                   string
-	ApiUsageMonitoringDefaultClientTrackingPattern string
```

Comment:
This should be considered as breaking API change and requires planning. Please add a warning log, if used instead of deleting it.